src/redux/villagers/villagers.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+
+const GET_VILLAGERS = 'GET_VILLAGERS';
+const GET_A_VILLAGER = 'GET_A_VILLAGER';
+const GET_VILLAGER_LENGTH = 'GET_VILLAGER_LENGTH';
+const API_URL = 'https://acnhapi.com/v1a/villagers/';
+
+const initialState = [];
+export const getVillagers = () => async (dispatch) => {
+  await axios.get(API_URL, {
+    method: 'GET',
+  })
+    .then((response) => {
+      dispatch({
+        type: GET_VILLAGERS,
+        payload: response,
+      });
+    });
+};
+
+export const getVillagerLength = () => async (dispatch) => {
+  await axios.get(API_URL, {
+    method: 'GET',
+  })
+    .then((response) => {
+      dispatch({
+        type: GET_VILLAGER_LENGTH,
+        payload: response,
+      });
+    });
+};
+
+export const getOneVillager = (villagerID) => async (dispatch) => {
+  await axios.get(`https://acnhapi.com/v1a/villagers/${villagerID}`, {
+    method: 'GET',
+  })
+    .then((response) => {
+      dispatch({
+        type: GET_A_VILLAGER,
+        payload: response,
+      });
+    });
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case GET_VILLAGERS:
+      /* eslint-disable no-unused-vars */
+      return action.payload.data.sort((a, b) => 0.5 - Math.random());
+      /* eslint-enable no-unused-vars */
+    case GET_A_VILLAGER:
+      /* eslint-disable dot-notation */
```

Comment:
- [ ] Kindly enable the eslint for **`no-unused-vars`** as I have earlier suggested
- [ ] Kindly remove the eslint-disable for **`dot-notation`**. It is redundant, as your code work just fine without disabling it.
![sunny-dot](https://user-images.githubusercontent.com/36542986/142586343-364afd40-40be-4a1f-a673-3a721aedefa5.jpg)